walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,39 @@
 
 package com.google.sps;
 
+import com.google.sps.FindMeetingQuery;
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+    //no Options For Too Long Of A Request
+    if( request.getDuration() > TimeRange.WHOLE_DAY.duration() ){
```

Comment:
Remove the space before the request and add a space after the if

`if (request.getDuration() ... ) {`